interface/modules/custom_modules/oe-module-ehi-exporter/table.sql

```diff
@@ -0,0 +1,65 @@
+#used for cleaning up the tables if needed
+#DROP TABLE IF EXISTS ehi_export_job_task_patients;
+#DROP table IF EXISTS ehi_export_job_tasks;
+#DROP TABLE IF EXISTS ehi_export_job_patients;
+#DROP TABLE IF EXISTS ehi_export_job;
+
+#IfNotTable
+CREATE TABLE `ehi_export_job`(
+   `ehi_export_job_id` int(11) NOT NULL AUTO_INCREMENT,
+   `uuid` BINARY(16) DEFAULT NULL,
+   `user_id` BIGINT(20) NOT NULL COMMENT 'FK to users.id - represents the user that started the export process',
+   `creation_date` DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+   `completion_date` DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+   `status` VARCHAR(20) NOT NULL DEFAULT 'processing' COMMENT 'processing=export in progress, failed=error occurred in one or more tasks, completed=export completed without errors',
+   `include_patient_documents` TINYINT(1) NOT NULL DEFAULT 1 COMMENT '1=Include patient documents into zip export, 0=no patient documents included in the export',
+   `document_limit_size` INT(11) UNSIGNED NOT NULL DEFAULT 524288000 COMMENT 'The batch zip file export size, default set to 500MB max value size is 4GB',
+   PRIMARY KEY(`ehi_export_job_id`),
+   CONSTRAINT `FK_ehi_export_job_users` FOREIGN KEY (`user_id`) REFERENCES `users`(`id`) ON DELETE RESTRICT ON UPDATE RESTRICT
+) ENGINE = INNODB COMMENT = 'User initiated export of patient electronic health info.';
+#EndIf
+
+#IfNotTable
+CREATE TABLE `ehi_export_job_patients`(
+     `ehi_export_job_id` int(11) NOT NULL,
+     `pid`  BIGINT(20) NOT NULL,
+     -- join tables should always cascade
+     PRIMARY KEY(`ehi_export_job_id`, `pid`),
+     CONSTRAINT `FK_ehi_export_job_patients_job_id` FOREIGN KEY (`ehi_export_job_id`) REFERENCES `ehi_export_job`(`ehi_export_job_id`) ON DELETE CASCADE ON UPDATE CASCADE,
+     CONSTRAINT `FK_ehi_export_job_patients_patient_id` FOREIGN KEY (`pid`) REFERENCES `patient_data`(`pid`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE = INNODB COMMENT = 'Patients to be exported in the ehi_export_job';
+#EndIf
+
+#IfNotTable
+CREATE TABLE `ehi_export_job_tasks`(
+     `ehi_task_id` BIGINT(20) NOT NULL AUTO_INCREMENT,
+     `ehi_export_job_id` int(11) NOT NULL,
+     `export_document_id` int(11) DEFAULT NULL COMMENT 'FK to documents.id - represents the document result file that was created as part of this task',
+#      `export_document_size_limit` int(11) DEFAULT 525312500 COMMENT 'The patient documents export size, default set to 500MB',
+     `creation_date` DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+     `completion_date` DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+     `status` VARCHAR(20) NOT NULL DEFAULT 'pending' COMMENT 'pending=task export has not started, processing=task export in progress, failed=error occurred in the task, completed=export task completed without errors',
+     `error_message` TEXT DEFAULT NULL COMMENT 'The error that occurred in the export process, only populated if status=failed',
```

Comment:
remove above 'DEFAULT NULL'

Reply:
Should be resolved now.

Reply:
still needs to be addressed (remove the above `DEFAULT NULL`)

Reply:
still needs to be addressed (remove the above DEFAULT NULL)